Translation/en/Text/Override.txt

```diff
@@ -390,54 +346,54 @@ sr:"速度([\=\+\d\.\%]*)"=Mov Speed $1
 炼丹成丹值+25=Alchemy Success Rate +25
 火焰掌控者=<size\=16>Alchemist of Flame</size>
 炼丹火力值+30=Alchemy Fire Value +30
-丹心凝练=<size\=16>Alchemist of Quality</size>
-炼丹融合度+9%=Alchemy Quality +9%
+丹心凝练=<size\=16>Fusion Master</size>

```

Comment:
Alchemist of Fusion would be better here primarily because, "Fusion Master" is Already used in lines 551 pre pull | 507 post poll.